tests/WebTest.php

```diff
@@ -81,7 +81,7 @@ public function it_shows_blog_articles(string $id) : void
 
         $this->assertSame(200, $response->getStatusCode());
         $this->assertSame('text/html; charset=UTF-8', $response->headers->get('Content-Type'));
-        $this->assertSame("Blog article {$id}", $crawler->text());
+        $this->assertSame("Blog article {$id}", trim($crawler->filter('body')->text()));
```

Comment:
good for now, will have to look into a certain element (`h1`?) when that is used

Reply:
Will become https://github.com/libero/browser/blob/1bc82c5dd9e95a9f3eaf49ff44f47b6e85aa3e8f/tests/WebTest.php#L83 in #25.